Packs/Slack/Scripts/SlackBlockBuilder/README.md

```diff
@@ -126,3 +126,18 @@ Verify your Slack blocks payload is valid. Try simplifying the payload. Test wit
 **Issue**: The survey is sent to Slack successfully, but clicking the Submit button in Slack does nothing. No response is returned to Cortex XSOAR. There may be a ⚠️ icon next to the Submit button in Slack. 
 
 **Troubleshooting**: Ensure the "Long running instance" checkbox is checked in the Slack v3 integration instance settings.
+
+---
+**Issue**: Migrating to pack version 3.3.0.
+
+**Troubleshooting**: # Implementation Guide for Changes in SlackV3 for XSOAR (Version 3.3.0)
+
+## 1. Review and Identify Affected Playbooks
+- Identify all playbooks currently utilizing the SlackBlockBuilder script or its derivatives.
+
+## 2. Implement Conditional Tasks
+- Introduce a conditional task in your playbooks that waits for the expected response.
```

Comment:
this is not the case in the original playbooks anyway?

Reply:
In the original, people were using polling tasks to check the context. I just wanted to be super clear.